### DISCUSSION TITLE
Discussion Regarding finding enough issues

### DISCUSSION THREAD

**Discussion by Raghav-45** [spacedriveapp/spacedrive#1142] [Community Support]
Hey @utkubakir and other maintainers,

I hope you're doing well. I wanted to reach out and ask for some help regarding finding enough issues to resolve. I've been looking through the repository, but it seems a bit challenging to find suitable ones. I want you and other peoples to create more issues which we can solve as Community.

Also, I noticed that some of the issues you created previously are missing images, making it difficult for me to understand the context. Could you please help me with this and other related issues?

Your assistance would be greatly appreciated. Thank you!

Best regards, Aditya

**Comment by utkubakir** [comment]
@Raghav-45 Yeah of course we can make more issues public. You can comment under the issue to request the image or further clarification on any issue anytime!
